sdk/monitor/Azure.Monitor.Ingestion/src/Generated/LogsIngestionClient.cs

```diff
@@ -153,7 +153,7 @@ internal virtual async Task<Response> UploadAsync(string ruleId, string streamNa
         /// ]]></code>
         /// </example>
         /// <remarks> See error response code and error response message for more detail. </remarks>
-        internal virtual Response Upload(string ruleId, string streamName, RequestContent content, string contentEncoding = "gzip", RequestContext context = null)
+        public virtual Response Upload(string ruleId, string streamName, RequestContent content, string contentEncoding = "gzip", RequestContext context = null)
```

Comment:
Do we Gzip the user content here? Or is this meant to be a method that will just take exactly what the user provides and send it to the service. If that is the case, I'm not sure that it makes sense to default the contentEncoding parameter as "gzip".

Reply:
We are trying to gzip by default, so any input provided by the user will be gzipped

Reply:
request gets constructed on 109

Reply:
Got it! So this method will still GZip for users based on the content-encoding.